src/index.js

```diff
@@ -89,7 +89,7 @@ function decode (bufOrString) {
  * Is the given data multibase encoded?
  *
  * @param {Buffer|string} bufOrString
- * @returns {boolean}
+ * @returns {boolean|string}
```

Comment:
It actually should just be boolean. It is the code that has a bug

Reply:
What is the dependency of `isEncoded`? if no, maybe change it into `return !!base.name`

Reply:
why is `Boolean(x)` considered better than `!!x`? Won't it introduce one more function call, hence slower? Or it avoid type conversion so being faster?

Reply:
- `Boolean(x)` is more explicit, and clearer to folks who don't write as much JS.
- `!!x` is idiomatic amougst JS users, mainly because it's fewer characrers to type and looks cool. 

I had assumed `!!` might be slightly faster, but https://jsperf.com/bool-not-not suggests there isn;t much in it, and if anything Boolean is slightly faster. Regardless, it's good to optimise for readability first, and optimize for performance when you have benchmarks of an entire app that shows where the bottlenecks are.